.github/workflows/ci.yml

```
@@ -72,6 +72,8 @@ jobs:
         PKG_CACHE_PATH: ${{ steps.node-pkg-cache.outputs.dir }}
     - name: Build package
       run: python build_package.py
+    - name: Install
+      run: python -m playwright install
```

```suggestion
      run: playwright install
```
Since we do `pip install -e .` it should be available in the PATH.

---

The fact that we pollute env with another executable in users's path makes me sad. I'd like to emphasize that playwright is a library, not an executable and recommend its use via `-m`. Once we gain a handful of `cli` capabilities, I'll be comfortable saying `playwright` in my cli!